fix: UTC-437: Fix prediction validation for Ranker

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/20278114710)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/699
<!-- Follow Merge -->

hakan458 committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20252979500

hakan458 committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20253543161

hakan458 committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20253627620

hakan458 committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20278114710

hakan458 committed:
Merge branch 'develop' into 'fb-utc-437'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20278114710

hakan458 commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/20319065681)
> Successfully merged:  create mode 100644 web/libs/ui/src/assets/icons/reset.svg

hakan458 committed:
Merge branch 'develop' into 'fb-utc-437'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20319065681

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/20344313567
